pkgs/by-name/du/dumphfdl/package.nix

```
@@ -0,0 +1,59 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  libconfig,
+  liquid-dsp,
+  fftwSinglePrec,
+  glib,
+  soapysdr-with-plugins,
+  sqlite,
+  zeromq,
+  gperftools,
+  libacars,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dumphfdl";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "szpajder";
+    repo = "dumphfdl";
+    tag = "v${version}";
```

using finalAttrs would be preferred in this case
```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "dumphfdl";
  version = "1.6.1";

  src = fetchFromGitHub {
    owner = "szpajder";
    repo = "dumphfdl";
    tag = "v${finalAttrs.version}";
```